pymongo/client_session.py

```diff
@@ -947,9 +947,18 @@ def _txn_read_preference(self):
             return self._transaction.opts.read_preference
         return None
 
+    def _start_serv_sess(self):
+        if isinstance(self._server_session, _EmptyServerSession):
+            self._server_session, old = (
+                self._client._topology.get_server_session(),
+                self._server_session,
```

Comment:
I think this would be more clear without using tuple assignment:
```
old = self._server_session
self._server_session = self._client._topology.get_server_session()
```

Reply:
Done.